openretina/cli/train.py

```diff
@@ -10,16 +10,29 @@
 
 from openretina.data_io.base import compute_data_info
 from openretina.data_io.cyclers import LongCycler, ShortCycler
+import platform
+import sys
+import mlflow
 
 log = logging.getLogger(__name__)
 
 
-@hydra.main(version_base="1.3", config_path="../../configs", config_name="hoefling_2024_core_readout_high_res")
-def main(cfg: DictConfig) -> None:
-    train_model(cfg)
+@hydra.main(
+    version_base="1.3",
+    config_path="../../configs",
+    config_name="hoefling_2024_core_readout_high_res",
+)
+def main(cfg: DictConfig) -> float | None:
+    score = train_model(cfg)
+    if cfg.objective_target is not None:
+        return score
+    return None
```

Comment:
Do you need this if? Why not just always return the score?
```suggestion
    return score
```

Reply:
Sure, score is already None in there is no objective target, as handled in train_model().